src/App.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { createAppContainer, createSwitchNavigator } from 'react-navigation'
+import NavigationService from './navigationService'
+import SignUp from './screens/SignUp'
+import Home from './screens/Home'
+import LoadingScreen from './screens/LoadingScreen'
+import * as firebase from 'firebase'
+import firebaseConfig from './configs/firebase'
+
+if (!firebase.apps.length) {
```

Comment:
I think anyone can understand this condition at a glance. However, if you try to read it "in English", it would be: "if not length" or "if the length if false", even though we understand this is *interpreted* as "if the length evaluates to false*.

It's not a big deal, but I think a more readable approach could be `if (firebase.apps.length === 0)` or `if (!firebase.apps)`.